CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+
+Everything we are working on, or considering, is kept in GitHub issues.
+
+## How to report a bug or request a feature
+
+If you are able to work with GitHub then please "Create an issue"
+
+Before creating a new issue check to see if the issue already exists. If not, then please do report it. If you have permissions to assign the issue to people and milestones then try to select suitable options. 
+
+If you are not comfortable working with GitHub, but would still like to contribute, then talk to us.  Details at: https://github.com/opendataservices
+
+# Contributing Code and Documentation
+
+In order to be clear with everyone involved about how code (and documentation) contributions are accepted and used, we use Contributor Licence Agreements (CLA). We only require CLA's to accept 'commits' to the repository. Bug reports, feature requests, etc do not require a CLA.
+
+## Contributor Licence Agreements (CLA)
+
+The agreements we use allow the contributor to retain copyright over their contributions, while also granting us the rights to re-licence the projects in future without having to track down each contributor and ask for their agreement.
+
+We also stipulate that, should we re-licence a project, then we will only do so under an approved Open Source Institute licence.  
+
+It also protects us, by requiring contributors to check and confirm that they own the copyright on the contributions they make - this can help keep everything clear when, for example, a contribution is made on employers time.
+
+What this doesn't give us is the right to go out and fight copyright infringements on a contributor's behalf. 
+
+Our agreements are built of the templates provided by http://contributoragreements.org/
+
+We have agreements for individuals and institutions.
+
+Before contributions to any of our projects are accepted, we require you to complete the appropriate Contribution Licence Agreement and submit it to us, following the instructions below.
+
+### Submission Instructions
+
+We have a [generic agreement](http://contributoragreements.org/u2s/bm1v4q5je) that covers all code projects in the [Open Data Services GitHub Organisation](https://github.com/opendataservices)
+
+If this is amenable to you, feel free to complete it. We will be automatically notified that you have submitted an agreement and will contact you (usually in the same day, but in no more than 3 working days)
+
+However, we're more than happy to talk, so if you want to get in touch before you sign anything, then please do. 
+
+You can contact us at code@opendataservices.coop
+
+Once we have reviewed and signed a copy of the agreement we will be able accept your contributions.
+
+## How we use branches in this repository
+
+* `master` represents our main development branch, which gets deployed to http://http://grantnav.threesixtygiving.org/
+* Other branches represent development work or bug fixes.
```

Comment:
We have release branches of the form iteration0X.Y - these are what get deployed onto the live server, rather than the master branch. (Also there's a duplicate http:// here)

I amend this section is that's useful.